build-config/conf/busybox.config

```diff
@@ -727,7 +776,7 @@ CONFIG_WATCHDOG=y
 # CONFIG_SYS_EEPROM_SYSFS_FILE is not set
 CONFIG_SYS_EEPROM_I2C_DEVICE=""
 CONFIG_SYS_EEPROM_I2C_ADDR=
-CONFIG_SYS_EEPROM_I2C_MEM_ADDR_BITS=8
+CONFIG_SYS_EEPROM_I2C_MEM_ADDR_BITS=0
```

Comment:
Since CONFIG_SYS_EEPROM is not set, this option is set to 0. because this value depends on CONFIG_SYS_EEPROM and CONFIG_SYS_EEPROM_DEVICE_I2C